src/module-1/debounce/index.js

```diff
@@ -1,5 +1,12 @@
 export const debounce = (fn, delay = 0) => {
-
+  let timeout = false;
+  return function() {
+    if (timeout) return;
+    fn.apply(this, arguments);
```

Comment:
please do not use `arguments`, it's deprecated. You can pass `...args` destructured array to your func and use them in `apply`